docs/src/docs/pykitops/cicd.md

```diff
@@ -0,0 +1,15 @@
+---
+description: Use KitOps ModelKits from a CI/CD tool's pipelines.
+---
+# Using KitOps ModelKits from a CI/CD Tool
+
+To get the most out of KitOps secure ModelKits, many teams and organizations create, push, or pull ModelKits as part of their CI/CD pipelines. For example, when a team finishes tuning of a model, a pipeline will create a ModelKit that includes the model, the tuning dataset used, the artifacts generated by the successful experiment run, and the results of that experiment run. 
+
+Typically this is done by installing the Kit CLI and running `kit` commands as tasks in the pipeline.
+
+The KitOps community has shared some of their CI/CD tool integrations:
+* [Dagger Modules](https://daggerverse.dev/mod/github.com/jozu-ai/daggerverse/kit@fc53d7802be6553298f4f54dd73cc1c1913d4ce6)
```

Comment:
This link is not good anymore, we should be using `https://github.com/kitops-ml/daggerverse`